src/fs.h

```diff
@@ -21,6 +21,15 @@ namespace fs = boost::filesystem;
 namespace fsbridge {
     FILE *fopen(const fs::path& p, const char *mode);
 
+    /**
+     * Helper function asserting that \p base is an absolute path and returning the result of boost::filesystem::absolute(p, base) then.
```

Comment:
In commit "Introduce fsbridge::AbsPathJoin(base, p)." (c0da7456833fac6d5613d97e09266cf54f13594f)

Would be good to drop "and returning the result of boost::filesystem...". Return value is already described below and the point of this function is to avoid the need for people to understand the extremely complicated boost function, which doesn't make any assumptions, by replacing with it a more straightforward join assuming at least one absolute path.